doc: Added information about Matter WP and OOP for Matter (#21313)
nrfconnect/sdk-nrf: doc/nrf/protocols/matter/getting_started/low_power_configuration.rst

* :ref:`Matter window covering sample <matter_window_covering_sample>`
* :ref:`Matter weather station application <matter_weather_station_app>`

> The additional materials and tools that can help you to optimize, estimate and measure the power consumption of your device are:

Reviewer: ```suggestion
The following additional materials and tools might help you to optimize, estimate, and measure the power consumption of your device:
```
